packages/jsdoc-plugins/lib/statics-augmentor/statics-augmentor.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+
+'use strict';
+
+/**
+ * Manually adds doclets for inherited static members.
+ * Jsdoc doesn't support inheritance of static members which is why this plugin is needed.
+ */
+class StaticsAugmentor {
+	/**
+	 * @param {Array.<Doclet>} doclets
+	 */
+	constructor( doclets ) {
+		this._data = doclets;
+	}
+
+	/**
+	 * Main method responsible for inheriting static members.
+	 */
+	augmentStatics() {
+		const childClasses = this._getDescendants();
+
+		for ( const child of childClasses ) {
+			const parents = child.augments;
+
+			for ( const parent of parents ) {
+				const staticMembers = this._getStaticMembers( parent );
+
+				for ( const member of staticMembers ) {
+					this._handleStaticMember( member, child );
+				}
+			}
+		}
+	}
+
+	/**
+	 * Finds doclets of child classes which need to have inherited static members added.
+	 * These have to be of kind `class` or `interface` and have a non empty `augments` array.
+	 *
+	 * @returns {Array.<Doclet>}
+	 * @private
+	 */
+	_getDescendants() {
+		return this._data.filter( doclet =>
```

Comment:
Definitely too complex arrow function. Use `{}`. You can also split the condition into meaningful parts. Like this:

```js
const isSomething = a || b;
const isSomethingElse = c && !d;

return isSomething || isSomethingElse;
```